tests/dialects/test_spark.py

```diff
@@ -233,6 +233,14 @@ def test_hint(self, logger):
             },
         )
 
+    def test_minus(self):
+        self.validate_all(
+            "SELECT * FROM db.table1 MINUS SELECT * FROM db.table2",
+            write={
+                "spark": "SELECT * FROM db.table1 EXCEPT SELECT * FROM db.table2",
+            }
+        )
```

Comment:
Small nit, it would be best if new tests are appended at the bottom of each test file.


Also, this test case should include Databricks and possibly Spark <-> Databricks transpilations as well.

 If we decide to generalize the `MINUS -> TokenType.EXCEPT` mapping across all Tokenizers, we can move the respective tests to `test_dialect.py`.

Reply:
Hey, I've moved the test to the bottom and added `databricks` to writes. Do we need more testcases?

Reply:
That's fine for now, I'll will move these cases to `test_dialect.py`  anyways once I get to clean the other dialects